share/revng/test/configuration/revng/end-to-end.yml

```diff
@@ -120,6 +120,20 @@ commands:
       # TODO: re-enable the following after fixing the bug in ModelToHeader that caused it to fail
       # revng check-decompiled-c "$$WORKDIR/decompiled.c" -I"$$WORKDIR" -m${POINTER_SIZE};
 
+  #
+  # Produce switch-to-statement artifact, as a preliminary test for MLIR-based decompilation
+  #
+  - type: revng.decompiled-c
+    from:
+      - type: revng-qa.compiled
+        filter: one-per-architecture
+      - type: revng.analyzed-model
+    suffix: /
+    command: |-
+      WORKDIR="$OUTPUT/check-decompiled-c";
+      mkdir -p "$$WORKDIR";
+      revng artifact --model "$INPUT2" --resume "$OUTPUT" switch-to-statements "$INPUT1"
```

Comment:
`WORKDIR` is unused. 🤔

Reply:
Leftover from copy-paste

Reply:
Ah no, I actually changed the test to put the result in `WORKDIR`, which is helpful for debug. I pushed for you to see. Tell me if you want to drop it, or leave it.

Reply:
What stops you from using `$OUTPUT/prepare-for-clift`? No need to save it again.

Reply:
Fair enough. Dropped this.